java/src/com/google/idea/blaze/java/run/fastbuild/BazelFastBuildTestEnvironmentCreator.java

```diff
@@ -76,6 +77,12 @@ private static File getStandardJavaBinary(String runfilesPath) {
         return file.toPath().resolve("bin/java").toFile();
       }
     }
+
```

Comment:
Can you modify the function comment to indicate that it also check this new setting?

Reply:
Sorry for the late reply thanks for taking a look really appreciate it. Sounds good, I'll update the comment to reflect that it can also come from the setting.

Reply:
updated the java doc

Reply:
The code currently prioritizes the default toolchain, I'm assuming this is fine. Please let me know if you think the user setting should take priority.

Reply:
that makes sense, thanks @dtaveras!